client/src/App.tsx

```diff
@@ -21,7 +24,31 @@ function App(): JSX.Element {
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard">
-                  <Dashboard />
+                  <Dashboard>
```

Comment:
Can use this wrapper component within each view so that we have cleaner routes? :)

Reply:
Applied